content/articles/how-cart-algorithms-are-substantial-in-speech-emotion-recognition/index.md

```diff
@@ -0,0 +1,110 @@
+---
+layout: engineering-education
+status: publish
+published: true
+url: /how-cart-algorithms-are-substantial-in-speech-emotion-recognition/
+title: How CART Algorithms are Substantial in Speech Emotion Recognition
+description: In this article, we will assess the use of CART algorithms in speech emotion recognition.
+author: lynn-njoki
+date: 2021-11-22T00:00:00-18:00
+topics: [Machine Learning]
+excerpt_separator: <!--more-->
+images:
+
+  - url: /engineering-education/how-cart-algorithms-are-substantial-in-speech-emotion-recognition/hero.jpg
+    alt: CART example image 
+---
+Unlike humans, who have the human ability to recognize emotion from speech, machines do not have the sense to analyze feelings. Speech emotional recognition is vital in understanding human emotions such as amusement, frustration, or disappointment.
+<!--more-->
+
+Emotions are vital in day-to-day life, especially in interpersonal human interaction. They are helpful both in intelligent and rational decisions. Unlike humans, who have the human ability to recognize emotion from speech, machines do not have the sense to analyze feelings. Speech emotional recognition is vital in understanding human emotions such as amusements, frustrations, or disappointment. Speech emotion recognition informs the computer on the physical and emotional state of an individual from a speech.
+
+A wide range of studies have focused on detecting a person's emotions. They include the use of computer games, audio surveillance, robots, others. Machine learning algorithms have been termed as more efficient in classification and prediction, which is vital in speech emotion recognition. However, there is a lot of information about the use of [CART](/engineering-education/decision-tree-in-python/) in speech emotion recognition that needs assessment which the study focuses on.
+
+The article focuses on assessing the substantial use of CART algorithms in speech emotion recognition. In achieving the purpose, the article reviews the speech emotion system, how CART works, and various reasons affiliated with its effectiveness.
+
+### The system of emotion recognition
+
+The system of speech emotion recognition is so like the pattern system. The system automatically identifies the human beings' emotions from a voice. The system is usually based on the signal of speech and emotion recognition methods. The system consists of five main steps;
```

Comment:
Rephrase the first sentence to better convey the comparison between speech emotion recognition and the pattern system